portal-api/src/v1/api.test.js

```diff
@@ -0,0 +1,119 @@
+import axios, { HttpStatusCode } from 'axios';
+import { HEADERS, ROLES, ALL_TEAM_IDS } from '@ukef/dtfs2-common';
+import { getTfmTeam } from './api';
+
+const { DTFS_CENTRAL_API_URL, DTFS_CENTRAL_API_KEY } = process.env;
+const headers = {
+  [HEADERS.CONTENT_TYPE.KEY]: HEADERS.CONTENT_TYPE.VALUES.JSON,
+  'x-api-key': DTFS_CENTRAL_API_KEY,
+};
+
+console.error = jest.fn();
+jest.mock('axios');
+
+describe('getTfmTeam', () => {
+  const invalidTeamIds = [
+    null,
+    undefined,
+    {},
+    [],
+    '',
+    'invalid',
+    `${ALL_TEAM_IDS[0]}invalid`,
+    ROLES.MAKER,
+    ROLES.CHECKER,
+    ROLES.ADMIN,
+    ROLES.PAYMENT_REPORT_OFFICER,
+    ROLES.READ_ONLY,
+  ];
+
+  const validTeamIds = [ALL_TEAM_IDS[0], ALL_TEAM_IDS[1], ALL_TEAM_IDS[2], ALL_TEAM_IDS[3], ALL_TEAM_IDS[4], ALL_TEAM_IDS[5], ALL_TEAM_IDS[6], ALL_TEAM_IDS[7]];
```

Comment:
```suggestion
  const validTeamIds = ALL_TEAM_IDS;
```

Unless i'm missing something